Makefile

```diff
@@ -0,0 +1,10 @@
+#this will execute the program 
+OBJ = obj
```

Comment:
Lets remove the spaces around `=` in case of Makefiles.

Reply:
sure